src/resolvers/get-model-from-vin.ts

```diff
@@ -0,0 +1,14 @@
+import { Make } from '../constants/make'
+import { Model, Models } from '../constants/model'
+
+export function getModelFromVin(make: Make, vin: string): Models | null {
+  switch (make) {
```

Comment:
I assume you are using a switch with one case in anticipation of it expanding? Otherwise, I think an if statement is clearer.

Reply:
I think that makes sense for few cases as well. I was just following the other examples but they have a lot more cases 👍